libevm/VMFactory.cpp

```diff
@@ -54,6 +55,7 @@ VMKindTableEntry vmKindsTable[] = {
 #endif
 #if ETH_HERA
     {VMKind::Hera, "hera"},
+    {VMKind::HeraPlus, "hera+"},
```

Comment:
Should this be `heraplus` on the command instead?

Reply:
Can be  :)